templates/spiders/xmlfeed.tmpl

```diff
@@ -13,12 +16,11 @@ class $classname(XMLFeedSpider):
         # "brand_wikidata": "$brand_wikidata",
         # "operator": "$operator",
         # "operator_wikidata": "$operator_wikidata",
-        # "extras": Categories.SHOP_XYZ.value
     }
 
     def parse_node(self, response, selector):
-        item = {}
-        #item["url"] = selector.select("url").get()
-        #item["name"] = selector.select("name").get()
-        #item["description"] = selector.select("description").get()
+        item = Feature()
+        # item["name"] = selector.select("name").get()
```

Comment:
Suggest changing this default example to `ref` rather than `name` because `ref` is more-or-less mandatory, but `name` isn't.